examples/3d/elixir_euler_eoc_test.jl

```diff
@@ -0,0 +1,60 @@
+
+using OrdinaryDiffEq
+using Trixi
+
+###############################################################################
+# semidiscretization of the compressible Euler equations
+
+equations = CompressibleEulerEquations3D(2.0)
+
+initial_condition = Trixi.initial_condition_eoc_test_coupled_euler_gravity
```

Comment:
Isn't this one exported (at least on `dev`)?

Reply:
You're right. Fixed in 2f7f549.